### DISCUSSION TITLE
Dropbox file version history

### DISCUSSION THREAD

**Discussion by nivek88** [samschott/maestral#494] [Q&A] [answered]
If I switch from the Dropbox app to Maestral, is it still possible to view and restore previous versions of files and folders in my Dropbox account using the webbrowser?
I couldn't find any info about that, but I guess the answer is: yes you can?!
Thank you for your answer(s)!

**Comment by samschott** [comment, marked as answer]
Yes, absolutely!

**Reply by nivek88** [reply]
Thank you for your reply! Appreciate it!

**Comment by samschott** [comment]
BTW, Maestral also provides CLI commands to list and restore old versions. See [`maestral revs`](https://maestral.app/cli/revs), [`maestral diff`](https://maestral.app/cli/diff) and [`maestral restore`](https://maestral.app/cli/restore).
